<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72, 72, 72);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Top Text</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLineEdit" name="le_bottomtext">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(214, 214, 214);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(0, 0, 0);

}

QLineEdit:focus {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(170, 170, 255);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(255,255,255);
}</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lb_wait">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 0);
font: 75 14pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Please wait....</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Meme</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLineEdit" name="le_toptext">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(214, 214, 214);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(0, 0, 0);

}

QLineEdit:focus {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(170, 170, 255);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(255,255,255);
}</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Bottom Text</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="box_meme">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: rgb(214, 214, 214);
    border: 2px solid rgb(52, 53, 64);
    color: rgb(0, 0, 0);
}

QComboBox QAbstractItemView {
    background-color: rgb(214, 214, 214);
    color: rgb(0, 0, 0);
}

</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="bt_generate">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(214, 214, 214);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(0, 0, 0);

}

QPushButton:pressed {
						
	background-color: rgb(85, 170, 127);
						border-radius: 10px;  /* Закругление краев */
						border: 2px solid rgb(52, 53, 64);

}</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QLabel" name="lb_memimage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Mem Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="bt_clear">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
						border-radius: 10px;  /* Закругление краев */
						background-color: rgb(214, 214, 214);
						border: 2px solid rgb(52, 53, 64);
						color: rgb(0, 0, 0);

}

QPushButton:pressed {
						
	background-color: rgb(85, 170, 127);
						border-radius: 10px;  /* Закругление краев */
						border: 2px solid rgb(52, 53, 64);

}</string>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
